main.js

```diff
@@ -159,3 +159,40 @@ form.addEventListener('submit', (e) => {
   document.getElementById('err-display').innerHTML = text;
   document.getElementById('err-display').style.border = '1px solid rgba(255, 0, 0, 0.9)';
 });
+
+// Local storage
+
+// Select all form inputs
+const formInputs = document.querySelectorAll('.input-cls');
+const formInputsArr = Array.from(formInputs);
+
+// Select form sbmit button
+const submitBtn = document.querySelector('.submit-btn');
+
+// select localstorage to check if there is already existing data
+const localData = localStorage;
+
+// Object to store our collected data
+const ContactForm = {
+  fullName: '',
+  email: '',
+  message: '',
+};
+
+// Check if there is data and if its there, fill the inputs with it
+if (localData.fullName || localData.email || localData.message) {
+  formInputsArr[0].value = localData.fullName;
+  formInputsArr[1].value = localData.email;
+  formInputsArr[2].value = localData.message;
+}
+
+// Event listener to submit buton to store the data to local storage before submitting on click
+submitBtn.addEventListener('click', () => {
+  ContactForm.fullName = formInputsArr[0].value;
+  ContactForm.email = formInputsArr[1].value;
+  ContactForm.message = formInputsArr[2].value;
+
+  localStorage.setItem('fullName', ContactForm.fullName);
```

Comment:
- [ ] I kindly suggest that you follow the project requirements by ensuring that on page reload and not on submit, the data the user has entered is saved to local storage

## EXAMPLE
```
submitBtn.addEventListener('input', () => {
const ContactForm  = {
  ContactForm.fullName = formInputsArr[0].value;
  ContactForm.email = formInputsArr[1].value;
  ContactForm.message = formInputsArr[2].value;
}
  localStorage.setItem(''form', ContactForm);
});

```